### ISSUE TITLE
Mass Email

### ISSUE THREAD

**Issue by Hossy** [petio-team/petio#250]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
As an admin, I would like the ability to mass email my users (one-off sends).  For example, if I need to tell them of a new feature I added to be able to request movies and TV shows. ;-)  Different from welcome emails.

**Describe alternatives you've considered**
Ombi (what I use now)

**Comment by dannyhanes** [comment]
Yep! I would LOVE the ability to send emails to my users. I recently did a server upgrade and had to take Plex offline for a bit and it would have been nice if I could send them a quick email with a downtime notice.
